### ISSUE TITLE
Building against GNU Termcap fails

### ISSUE THREAD

**Issue by weiss** [erlang/otp#7381]
**Describe the bug**
Building against [GNU Termcap][1] fails:

```
 CC     obj/x86_64-pc-linux-musl/opt/jit/prim_tty_nif.o
nifs/common/prim_tty_nif.c:46:12: fatal error: curses.h: No such file or directory
   46 |   #include <curses.h>
      |            ^~~~~~~~~~
compilation terminated.
make[4]: *** [x86_64-pc-linux-musl/Makefile:916: obj/x86_64-pc-linux-musl/opt/jit/prim_tty_nif.o] Error 1
make[4]: Leaving directory '/home/runner/build/eturnal/x64-musl/src/otp_src_26.0/erts/emulator'
make[3]: *** [/home/runner/build/eturnal/x64-musl/src/otp_src_26.0/make/run_make.mk:35: opt] Error 2
make[3]: Leaving directory '/home/runner/build/eturnal/x64-musl/src/otp_src_26.0/erts/emulator'
make[2]: *** [Makefile:45: opt] Error 2
make[2]: Leaving directory '/home/runner/build/eturnal/x64-musl/src/otp_src_26.0/erts'
make[1]: Leaving directory '/home/runner/build/eturnal/x64-musl/src/otp_src_26.0/erts'
make[1]: *** [Makefile:60: jit] Error 2
make: *** [Makefile:503: emulator] Error 2
Error: Process completed with exit code 2.
```

[1]: https://www.gnu.org/software/termutils/manual/termcap-1.3/termcap.html

**To Reproduce**
Run `./configure && make` with GNU Termcap being available instead of, e.g., ncurses.

**Expected behavior**
Building against GNU Termcap succeeds, as it did up to Erlang/OTP 25.x.

**Affected versions**
Erlang/OTP 26.0.

**Additional context**
We've linked (statically) against GNU Termcap in the past, as it's easier to build portable binaries than it is with ncurses.  However, I'm not sure whether supporting GNU Termcap is worth any hassle; or whether other non-ncurses implementations might be affected as well.  If you don't think so, just close this issue!

**Comment by garazdawi** [comment]
I think I've fixed it in #7533, can you check if it works for you?

**Comment by weiss** [comment]
> I think I've fixed it in #7533, can you check if it works for you?

Works fine, thank you very much!
